apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -1393,8 +1406,9 @@ def delete_port_postcommit(self, context):
             else:
                 self._delete_gw_port_nat_disabled(context)
         elif port.get('device_owner') == n_constants.DEVICE_OWNER_ROUTER_INTF:
-            self._perform_interface_port_operations(context, port, network,
-                                                    is_delete=True)
+            if self.fabric_l3:
```

Comment:
should the stuff for gateway/interface port be done only when fabric_l3 is True?